### PR TITLE
IAPage.js - update backend files extensions

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
Depends on #1318

### CODE REVIEW

**Comment by jdorweiler** [comment]
Want to add `md` and `txt` too?  I forgot those.

**Comment by MariagraziaAlastra** [comment]
@jdorweiler Aren't those text (data) files? Should we include them regardless?

File: src/js/ia/IAPage.js [hunk 1]
```diff
@@ -72,16 +72,18 @@
...
+                        front_end = ["handlebars", "css", "json"];
+                        back_end = ["pm", "t", "py", "sh", "rb", "pl"];
                         $.each(ia_data.live.code, function(idx) {
                             var file = ia_data.live.code[idx];
                             var type = file.replace(/.*\.([^\.]*)$/,'$1');
 
                             if ($.inArray(type, front_end) !== -1) {
                                 ia_data.live.front_end.push(file);
-                            } else if ($.inArray(type, back_end) !== -1) {
+                            } else if (($.inArray(type, back_end) !== -1) || (type === "js" && file.match(/\/lib\//))) {
```
Comment (nilnilnil) [inline src/js/ia/IAPage.js]: This probably deserves a comment. It looks like you're saying consider JavaScript files in the lib directory as backend files.

Comment (MariagraziaAlastra) [inline src/js/ia/IAPage.js]: @nilnilnil Thanks, I added a little comment :smile:

**Comment by MariagraziaAlastra** [comment]
@jdorweiler 
screenshot-maria duckduckgo com 5001 2016-04-11 15-13-25

**Comment by jdorweiler** [comment]
@MariagraziaAlastra :+1: 
I was thinking we want to include the README files for fathead/longtail.  They are generally txt or md files https://app.asana.com/0/inbox/11472677167854/109154791732210/110894475524736

Maybe it's best to restrict that to only fathead or longtail IA pages?

**Comment by MariagraziaAlastra** [comment]
@jdorweiler Maybe that should be a different section, then? Or should they go with the other files, under back-end?

**Comment by jdorweiler** [comment]
@MariagraziaAlastra I wouldn't be opposed to a separate section if it's not too much work to add.  Otherwise backend is fine too.

**Comment by MariagraziaAlastra** [comment]
@jdorweiler what do you think? :smile: 
screenshot-maria duckduckgo com 5001 2016-04-11 16-43-48

**Comment by MariagraziaAlastra** [comment]
@jdorweiler Only including files named "readme"
screenshot-maria duckduckgo com 5001 2016-04-11 17-50-46

**Comment by jdorweiler** [comment]
:+1:  thanks

**Pull request merged**